File: MongoDB Express - API - Orphanslife/Data/model/Orphanage.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema
const passportLocalMongoose = require('passport-local-mongoose');

var Orphanage = new Schema({
    orphanage_id:{
        type: Number
    },
    type:{
        type: String
    },
    orphanage_address:{
        type: String
    },
    orphanage_location_id:{
        type: Number
    },
    contact_person:{
        type: String
    },
    orphanage_mobile1:{
        type: String
    },
    orphanage_mobile2:{
        type: String
    },
    orphanage_email:{
        type: String
    },
    in_home:{
        type: Number
    },
    adoptable:{
        type: Number
    },
    boys:{
        type: Number
    },
    girls:{
        type: Number
    },
    orphanage_image:{
        type: Number
    },
    orphanage_created_at:{
        type: String
    },
    orphanage_updated_at:{
        type: String
    }
}, { collection: 'orphanage' })
  
Orphanage.plugin(passportLocalMongoose);
  
module.exports = mongoose.model('Orphanage', Orphanage)

File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/model/entity/Orphanage.kt
package com.kaligotla.orphanslife.model.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(
    tableName = "orphanage"
)

data class Orphanage(

    @PrimaryKey
    @SerializedName("orphanage_id")
    val orphanage_id: Int,

    @SerializedName("type")
    val type: String,

    @SerializedName("orphanage_address")
    val orphanage_address: String,

    @SerializedName("orphanage_location_id")
    val orphanage_location_id: Int,

    @SerializedName("contact_person")
    val contact_person: String,

    @SerializedName("orphanage_mobile1")
    val orphanage_mobile1: String,

    @SerializedName("orphanage_mobile2")
    val orphanage_mobile2: String,

    @SerializedName("orphanage_email")
    val orphanage_email: String,

    @SerializedName("in_home")
    val in_home: Int,

    @SerializedName("adoptable")
    val adoptable: Int,

    @SerializedName("boys")
    val boys: Int,

    @SerializedName("girls")
    val girls: Int,

    @SerializedName("orphanage_image")
    val orphanage_image: String,

    @SerializedName("orphanage_created_at")
    val orphanage_created_at: String,

    @SerializedName("orphanage_updated_at")
    val orphanage_updated_at: String,
)
